tp1/tp1.go

```diff
@@ -1,12 +1,32 @@
 package tp1
 
+import (
+	"sort"
+	"strconv"
+)
+
+type producto struct {
+	supermercado string
+	id           int
+	precio       int
+}
+
 // Producto contiene metodos que nos permiten acceder
 // a atributos que esperamos de un Producto.
 type Producto interface {
 	ID() int
 	Precio() int
 }
 
+func (p producto) ID() int {
+	return p.id
+}
+
+func (p producto) Precio() int {
+
```

Comment:
cuando instalamos go nos viene incluido con una herramienta que se llama 'gofmt'. Nos permite formatear el codigo y que todo codigo Go se vea igual. Hay integraciones para los editores que te lo formatean on save. Recomiendo que lo uses 😄

Reply:
Buenas tardes matias, al querer correr el test me tira un error en esta linea:
.\tp1_test.go:49:51: invalid use of ... in call to test.productos.CalcularPrecios
tengo que modificarle algo?